cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -52,6 +53,8 @@ class YamlConfig(
     private val storageLocation = "us-central1"
 
     fun getGcsBucket() : String {
+        if (bitrise) return rootGcsBucket
```

Comment:
we probably want something like `if (FtlConstants.useMock)` instead of `if (bitrise)`

bitrise should use the same logic as local runs